src/test/java/tests/config_app_tests/award_programs_tests/SwapAwardProgramsTest.java

```diff
@@ -0,0 +1,56 @@
+package tests.config_app_tests.award_programs_tests;
+
+import business_objects.UserBO;
+import business_objects.UserFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import page_objects.config_app_pages.common_config_app_pages.IafHomePage;
+import tests.common_test.BaseTest;
+import page_objects.config_app_pages.award_program_pages.AwardProgramsPage;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static business_objects.TestClients.CLIENT_5021;
+import static service_classes.login_service.LoginService.loginToIaf;
+
+public class SwapAwardProgramsTest extends BaseTest {
+
+    UserBO iafConsoleUser = UserFactory.createIafConsoleUser();
+
+    @Test(description = "Successful swapping of award programs for 5021 client", priority = 0)
+    public void swapAwardProgramsTest() throws InterruptedException {
+        loginToIaf(getWebDriver(), iafConsoleUser);
+        new IafHomePage(getWebDriver())
+                .openConfigApp()
+                .clickClientLink(CLIENT_5021.getConfigAppName())
+                .moveToAwardPrograms();
+
+        String textInitial = getWebDriver().findElement
+                (By.xpath("//tbody[@class='yui-dt-data']/tr[1]//a")).getText();
```

Comment:
We should avoid of findElement() usage within test methods. Please create appropriate method for corresponding page and re-use it in needed tests

Reply:
Fixed